### PR TITLE
Added South Korean (ko-KR) Internationalization Support

### PR DESCRIPTION
**Pull request by dance2die**
# ❓ What?
Added internationalization support for South Korean language, `ko-KR`.

# 🤔 Why?
As I recommend http://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/ to fellow Korean developers, it's sometimes easier to use common Korean terminology to refer to React life cycle methods and phases.

# 🚀 Demo

### CODE REVIEW

File: src/i18n/json/ko-KR.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
[the remark this answers is not in the document]
Comment (dance2die) [reply src/i18n/json/ko-KR.json]: Thank you @wojtekmaj.
`ko-KR.` is missing 3 fields from `pl-PL.json`.
I will update the PR.

**Comment by dance2die** [comment]
I've add missing properties in Korean language file, `ko-KR.json` from `pl-PL.json`.

# Property Comparison

image

# Changes Reflected on the Site

image

**Comment by wojtekmaj** [comment]
Thank you!

This PR is 💯  :D Very well done :)

**Comment by dance2die** [comment]
You're welcome and also thank you for the wonderful site.
Been visiting often to refer to Lifecycle methods often 😄

**Pull request merged**